extensions/blocks/membership-button/edit.jsx

```diff
@@ -267,6 +267,31 @@ class MembershipsButtonEdit extends Component {
 		</div>
 	);
 
+	renderDisclaimer = () => {
+		return (
+			<div className="membership-button__disclaimer">
+				{ __( 'Read more about ', 'jetpack' ) }
```

Comment:
This is going to be very hard to translate since it's split into different strings. Since the 2 links lead to the same page, what do you think about having one single string leading to the support page?

`<a>{ __( 'Read more about memberships and related fees.', 'jetpack' ) }</a>`

Reply:
Yeah, I agree. Done!